data/meson.build

```diff
@@ -94,15 +94,22 @@ if build_daemon
       dynamic_options += 'RestrictAddressFamilies=AF_NETLINK AF_UNIX'
     endif
 
-    # the order of SystemCallFilter is important as the first encountered will take precedence and
-    # will dictate the default action
-    syscall_allow = ['@system-service']
-    syscall_deny = ['@resources']
+    # allow-list, see `systemd-analyze syscall-filter` for what each group adds
+    syscall_filter = [
+      '@basic-io',
+      '@file-system',
+      '@io-event',
+      '@ipc',
+      '@network-io',
+      '@process',
+      '@sync',
+      'ioctl',
+      'uname',
+    ]
     if allow_flashrom
-      syscall_allow += ['@raw-io']
+      syscall_filter += ['@raw-io']
```

Comment:
Shouldn't raw io NOT be filtered when allow flashrom?

Reply:
It's an allow-list only with this PR.

Reply:
How counterintuitive how this stuff works. Ok.